build_openroad.sh

```diff
@@ -273,6 +273,11 @@ __local_build()
             set -u
         fi
 
+        if [[ -d "tools/yosys/abc/.git" ]]; then
+            # update indexes to make sure git diff-index uses correct data
+            git -C tools/yosys/abc update-index --refresh
```

Comment:
ancient versions of git doesn't have the -C option, so this would be a problem on CentOS 7. 

@maliberty Do you care about CentOS 7 anymore?

Reply:
We are in the process of leaving centos7 behind but have no fully gotten there yet.

Reply:
`-C ...` changed to `--work-tree` and `--git-dir`